src/Display/Bcm/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ add_library(DisplayBcm ${source_files})
 
 target_include_directories(DisplayBcm PUBLIC include)
 
-include_directories(${BCM_INCLUDE_DIR})
+target_include_directories(DisplayBcm PUBLIC ${BCM_INCLUDE_DIR})
```

Comment:
Not quite sure on the difference between include_directories and target_include_directories, The latter would be specific to DisplayBcm module?